rest-api-spec/src/main/resources/rest-api-spec/api/tasks.get.json

```diff
@@ -8,6 +8,7 @@
       "parts": {
         "task_id": {
           "type": "string",
+          "required" : true,
```

Comment:
This is not actually required, you can do `GET /_tasks` no problem

Reply:
@dakrone you are correct! But when you call `GET /_tasks` without `task_id`, it is mapped to :  https://github.com/elastic/elasticsearch/blob/3deba0ed1fe43faa9d22a0e924a1c6f8e60df3ce/core/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestListTasksAction.java#L52-L56 defined by `tasks.list` in the `rest-api-spec`s.
 
`tasks.get` is for :  https://github.com/elastic/elasticsearch/blob/3deba0ed1fe43faa9d22a0e924a1c6f8e60df3ce/core/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestGetTaskAction.java#L37-L40

Reply:
That's true, and there is the other .json spec file that mentions solely `/_tasks`

Hmm... I'm not sure what the best thing for this situation is, perhaps leaving it as `required` is the best since it only pertains to this particular API (not `tasks.list.json`)

Reply:
are there downsides for this change in the spec? I think it makes sense in the context of the get task API.

Reply:
@javanna yeah I think it does too, I just wanted to think about it for a little bit. I'll merge this.